packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -440,10 +440,16 @@ class OpPerfTelemetry {
 }
 export interface IPerfSignalReport {
 	/**
-	 * Identifier for the signal being submitted in order to
-	 * allow collection of data around the roundtrip of signal messages.
+	 * Identifier for the signal being submitted in order
 	 */
 	signalSequenceNumber: number;
+
+	/**
+	 * Identifier for broadcast signals being submitted in order to
+	 * allow collection of data around the roundtrip of signal messages.
+	 */
+	broadcastSignalSequenceNumber: number;
```

Comment:
You are using this as a counter rather than a sequence number really. We can call it a count.